runtime/wasm/src/host_exports.rs

```diff
@@ -78,7 +78,7 @@ impl<C: Blockchain> HostExports<C> {
             subgraph_id,
             api_version: data_source.mapping().api_version.clone(),
             data_source_name: data_source.name().to_owned(),
-            data_source_address: data_source.source().address.clone(),
+            data_source_address: data_source.address().unwrap_or_default().to_owned(),
```

Comment:
Why is it fine to have an empty `Vec` right here?

Reply:
This is only used when being returned from the `data_source_address` host export, which used `unwrap_or_default`. So I figured we might as well do that earlier. But it is not supposed to be a functional change.